internal/tui/model_test.go

```
@@ -0,0 +1,751 @@
+	return controlplane.ActionResult{}, nil
+}
+func (staticService) Subscribe(context.Context) <-chan controlplane.Event {
+	return make(chan controlplane.Event)
+}
+func (staticService) Cancel(context.Context, string) error { return nil }
+
+func TestModelComposerCapturesObjectiveForRun(t *testing.T) {
+	model := NewModel(controlplane.DefaultCatalog(), Options{Motion: false})
+	model.Update(tea.WindowSizeMsg{Width: 80, Height: 24})
+
+	var updated tea.Model = model
```

golangci-lint (2.13.2)

[error] ineffectual assignment to updated

(ineffassign)